src/core/algorithms/ind/ind.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace model {
+
+using AttributeIndex = unsigned int;
```

Comment:
We have `Column::IndexType` for this (also see https://github.com/Mstrutov/Desbordante/issues/273)

Reply:
Although maybe `Column::IndexType` should be moved out of `Column` class, since we don't really want to include `column.h` here.
Nevertheless, we certainly don't want another alias which represents the exact same thing -- type of index of the column in the table, so please fix this somehow :)

Reply:
We already have `config::IndexType` in `src/core/config/indices/type.h` and `TablePos` in `src/core/algorithms/fd/hycommon/types.h`...